ericman.sh

```diff
@@ -83,6 +83,20 @@ function eric_bundle {
       --exclude ".install" \
       "${b}/" "${output}/"
 
+# we are converting all line breaks of the xsd files with dos2unix to LF because it is inconsistent in eric (sometimes LF, sometimes CRLF)
+    if type -p dos2unix; then
+      echo "running dos2unix for all .xsd files in output folder"
+      for any_file in "${output}/"*.xsd; do
```

Comment:
I'm pretty sure this does not capture all files you are looking for. You probably want `shopt -s globstar` and go with `${output}/**.xsd`?